libs/device-core/src/commands/use-cases/app-backup/restoreAppStorageInit.ts

```diff
@@ -0,0 +1,85 @@
+import Transport, { StatusCodes, TransportStatusError } from "@ledgerhq/hw-transport";
+import { LocalTracer } from "@ledgerhq/logs";
+import type { APDU } from "../../entities/APDU";
+
+/**
+ * Name in documentation: INS_APP_STORAGE_RESTORE_INIT
+ * cla: 0xe0
+ * ins: 0x6c
+ * p1: 0x00
+ * p2: 0x00
+ * data:
+ *  - LC: BACKUP_LEN_LEN (=0x04) + APP_NAME_LEN (1 byte)
+ *  - DATA: BACKUP_LEN + APP_NAME
+ *
+ * For example, the 'bitcoin' app with backup of length 0x00007000:
+ *    1. LC is 0x0b
+ *    2. DATA is 0x00007000 0x626974636f696e
+ */
+const RESTORE_APP_STORAGE_INIT: APDU = [0xe0, 0x6c, 0x00, 0x00, undefined];
+
+/**
+ * 0x9000: Success.
+ * 0x5123: Application not found.
+ * 0x662F: Invalid device state, recovery mode.
+ * 0x5501: Invalid consent, user rejected.
+ * 0x5502: Invalid consent, pin is not set.
+ * 0x670A: Invalid application name length, two chars minimum.
+ * 0x6733: Invalid BACKUP_LEN value.
+ */
+const RESPONSE_STATUS_SET: number[] = [
+  StatusCodes.OK,
+  StatusCodes.APP_NOT_FOUND_OR_INVALID_CONTEXT,
+  StatusCodes.CUSTOM_IMAGE_BOOTLOADER,
```

Comment:
[SHOULD] same remark as for other commands, define another StatusCode even if the value is the same